Source/WebGPU/WebGPU/Texture.mm

```diff
@@ -2150,13 +2150,13 @@ static bool textureViewFormatCompatible(WGPUTextureFormat format1, WGPUTextureFo
     NSUInteger blockWidth = Texture::texelBlockWidth(format);
     if (!blockWidth) {
         ASSERT_NOT_REACHED();
-        return 0;
+        return NSUIntegerMax;
     }
-    if (textureWidth % blockWidth) {
-        ASSERT_NOT_REACHED();
-        return 0;
-    }
-    NSUInteger blocksInWidth = textureWidth / blockWidth;
+    NSUInteger add = 0;
+    if (textureWidth % blockWidth)
+        add = 1;
+
+    NSUInteger blocksInWidth = textureWidth / blockWidth + add;
```

Comment:
this doesn't need checked arithmetic because `add` is only non-zero when `blockWidth > 1`, so `textureWidth` is at most uint32_t::max / 2